pkg/meta/conditions.go

```diff
@@ -30,6 +30,11 @@ const (
 	// ReleasedCondition represents the status of the last release attempt
 	// (install/upgrade/test) against the latest desired state.
 	ReleasedCondition string = "Released"
+
+	// ReversingCondition is the name of the Reversed condiiton
+	// This captures the state of reversing the forward workflow
+	// on deletion of the ApplicationGroup
+	ReversingCondition string = "Reversing"
```

Comment:
Why do we have both a reversing condition and a reversing reason? It seems that we are only using the `ReversingReason` and not the `ReversingCondition`?

Reply:
I wasnt sure how this worked lol. So should I remove ReversingCondition?

Reply:
Yes, we are not using this, so just the reason is fine